x/supply/types/query.pb.go

```diff
@@ -6,6 +6,7 @@ package types
 import (
 	context "context"
 	fmt "fmt"
+	github_com_cosmos_cosmos_sdk_types "github.com/cosmos/cosmos-sdk/types"
 	types "github.com/cosmos/cosmos-sdk/types"
```

Comment:
this is a bit strange that the types are duplicated? and why is the supply query updated?

Reply:
Because this PR changes the protobuf code generation logic (it is now exactly same as `cosmos-sdk`).